book/quick-brick/native-project-dev.md

```diff
@@ -0,0 +1,132 @@
+# Get ready to work on Quick-Brick
+
+1. [Description](#description)
+2. [Create run a quick-brick bundle from localhost](#localhost_qb)
+3. [iOS/tvOS Quick-Brick](#ios_qb)
+4. [Android and Android T.V](#android_qb)
+
+***
+
+## Description {#description}
+
+This guide will provide information about how to test your react native quick-brick bundle on a native simulator/emulator or actual devices.
+
+There are two options to run a quick-brick bundle on a native device.
+
+* [@applicaster/zapplicaster-cli](https://github.com/applicaster/QuickBrick/blob/master/packages/zapplicaster-cli/README.md) - this is the offical tool Applicaster provide for developing, testing and running an app on native devices.
+
+* Manual build - you can create a native project without using the Applicaster CLI tool. With this you will understand the steps needed to create a native project.
```

Comment:
You can create a project without the applicaster cli, but you won't have access to the native packager.
Besides, I don't think the second sentence is correct. Developers who want to create native plugins will still need - in some cases - to have their QB workspace ready, as many types of native plugins will require a react-native front-end to be consumed in the quick brick app